sql/src/main/java/org/apache/druid/sql/avatica/DruidMeta.java

```diff
@@ -104,8 +104,10 @@ public void openConnection(final ConnectionHandle ch, final Map<String, String>
   {
     // Build connection context.
     final ImmutableMap.Builder<String, Object> context = ImmutableMap.builder();
-    for (Map.Entry<String, String> entry : info.entrySet()) {
-      context.put(entry);
+    if (info != null) {
```

Comment:
i wonder if this was causing some problem before... looking at other implementations some do check for null on this field, :+1: